### PR TITLE
created variables, functions & calls as per assignment

### PR DESCRIPTION
**Pull request by efm-productions**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -57,9 +61,49 @@ void variableDeclarations()
...
+    ignoreUnused(outsideTemp);
+    ignoreUnused(studentGPA);
+    ignoreUnused(interestRate);
+    ignoreUnused(percentComplete);
+    ignoreUnused(furnaceTemp);
+    ignoreUnused(discountAmount);
+    ignoreUnused(lotteryJackpot);
+    ignoreUnused(accountInArrears);
+    ignoreUnused(collegeDegree);
+    ignoreUnused(roomOccupied);
```
Comment (matkatmusic) [inline main.cpp]: Use a single call of `ignoreUnused`, like you're doing in the functions below.
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -131,27 +213,49 @@ int main()
...
+
     //6)
-    
+    auto invoiceTotalDue =  generateInvoice (248.16); 
+
     //7)
-    
+    char carOwner[25] = "John Smith";
+    char carMake[25] = "Buick";
+    auto reservedSpaceNumber = reserveAParkingSpace(carOwner, carMake, 16.f);
```
Comment (matkatmusic) [inline main.cpp]: No raw character arrays yet!
- [x] remove this.

File: main.cpp [hunk 3]
```diff
@@ -74,43 +118,81 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+double generateInvoice (double totalPrice, double salestaxRate = 4.5)
+{
+    ignoreUnused(totalPrice, salestaxRate);
+    return {};
+}
 /*
  7)
  */
-
+int reserveAParkingSpace(char custName[25], char carMake[25], float rentalHours, double hourlyRate = 25.00)
```
Comment (matkatmusic) [inline main.cpp]: No raw character arrays yet.  
- [x] revise.

File: main.cpp [hunk 4]
```diff
@@ -131,27 +213,49 @@ int main()
...
+    */
+    ignoreUnused(flightBooked);
+    ignoreUnused(errorCode);
+    ignoreUnused(accountBalance);
+    ignoreUnused(ConvertedTempValue);
+    ignoreUnused(invoiceTotalDue);    
+    ignoreUnused(reservedSpaceNumber);
+    ignoreUnused(interest);
+    ignoreUnused(result);
+
```
Comment (matkatmusic) [inline main.cpp]: call `ignoreUnused` once with all of your unused variables.
- [x] revise.

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (efm-productions) [reply main.cpp]: resvision completed

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**